temporal/api/temporal.api

```diff
@@ -0,0 +1,21 @@
+public final class com/juul/tuulbox/temporal/DateTimeFlowKt {
+	public static final fun instantFlow ()Lkotlinx/coroutines/flow/Flow;
+	public static final fun instantFlow (Lkotlinx/coroutines/CoroutineScope;Lkotlinx/coroutines/flow/SharingStarted;)Lkotlinx/coroutines/flow/StateFlow;
+	public static final fun localDateFlow ()Lkotlinx/coroutines/flow/Flow;
+	public static final fun localDateFlow (Lkotlinx/coroutines/CoroutineScope;Lkotlinx/coroutines/flow/SharingStarted;)Lkotlinx/coroutines/flow/StateFlow;
+	public static final fun localDateTimeFlow ()Lkotlinx/coroutines/flow/Flow;
+	public static final fun localDateTimeFlow (Lkotlinx/coroutines/CoroutineScope;Lkotlinx/coroutines/flow/SharingStarted;)Lkotlinx/coroutines/flow/StateFlow;
+}
+
+public final class com/juul/tuulbox/temporal/TemporalFlowKt {
+	public static final fun temporalFlow (Lkotlinx/coroutines/CoroutineScope;Lkotlinx/coroutines/flow/SharingStarted;Lkotlin/jvm/functions/Function0;)Lkotlinx/coroutines/flow/StateFlow;
+}
+
+public final class com/juul/tuulbox/temporal/TickerKt {
+	public static final fun ticker-ge6A_vg (DLkotlin/jvm/functions/Function0;)Lkotlinx/coroutines/flow/Flow;
```

Comment:
Is name mangling (while still on the public API) due to `internal` w/ `@PublishedApi`?

Reply:
Exactly. I can un-inline things if you'd prefer? These flows really don't need to be highly optimized.